src/media/modules/allNodes.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+
+namespace WPML\PB\Elementor\Media\Modules;
+
+
+class allNodes extends \WPML_Elementor_Media_Node {
+
+	/**
+	 * @param array $settings
+	 * @param string $target_lang
+	 * @param string $source_lang
+	 *
+	 * @return array
+	 */
+	public function translate( $settings, $target_lang, $source_lang ) {
+		$backgrounds = [
+			'_background_image',
+			'background_image',
+			'background_overlay_image',
+			'background_hover_image'
+		];
+
+		foreach ( $backgrounds as $background ) {
+			if ( isset ( $settings[ $background ]['url'] ) ) {
```

Comment:
I think this check is not needed because it's already done inside `translate_images_property`.

Reply:
In my previous code - it had to be there, but now my new code is not doing its job and I am checking on why.

Reply:
It is fixed now, the line is removed.